source/config-file/config-file-options.txt

```diff
@@ -59,6 +59,22 @@ You can configure the following settings in a configuration file:
      - Configure |compass| to not perform outgoing network operations other 
        than those to the database.
 
+Example 
+-------
+To view a sample |compass-short| configuration file, run the following command 
+in the folder containing your |compass| executable: 
```

Comment:
Question, do you have to be in the compass directory or will this work so long as the compass executable is in your path? 

If it works anywhere, consider: 
run the following command from a directory in your |compass| executable PATH.


(Aside - Do the compass docs normally refer to "folders" or "directories"? )

Reply:
Good catch — upon testing, it works anywhere. Will update the copy!
As for whether we refer to "folders" or "directories" it _does_ seem a bit inconsistent in the Compass docs for what term we use. Typically, it looks like we generally use "directory" but specifically for the directory with the Compass executable, the docs tend to refer to "folder".